src/sentry/web/frontend/setup_wizard.py

```diff
@@ -60,28 +60,34 @@ def get(self, request: Request, wizard_hash) -> HttpResponse:
             member_set__role__in=[x.id for x in roles.with_scope("org:read")],
             member_set__user_id=request.user.id,
             status=OrganizationStatus.ACTIVE,
-        ).order_by("-date_added")[:50]
+        ).order_by("-date_added")
 
-        filled_projects = []
+        projects = Project.objects.filter(organization__in=orgs, status=ObjectStatus.ACTIVE)
```

Comment:
nit: the usage of the `in` operator might become a bottleneck since the database will have to scan each row but at least now we have only 3 queries, which is significantly better.

Reply:
If you want to hardcore optimize the query you could leverage an in-memory temporary table with the ids you are looking for and joining on those but this is something that might not be needed for this use case.

Reply:
It's a great point and definitely an improvement we can think about when we observe performance problems. However, for the time being, I'd like to stick with this version and merge it. We can always revisit later.

Reply:
Yes, green light from my side for this. Thanks for taking the time to improve it!